modules/openapi-generator/src/main/resources/python/api_client.mustache

```diff
@@ -393,15 +396,15 @@ class ApiClient(object):
             then the method will return the response directly.
         """
         if not async_req:
-            return self.__call_api(resource_path, method,
+            return self.__call_api{{#detectAmznApiGatewayExtensions}}_signed{{/detectAmznApiGatewayExtensions}}(resource_path, method,
```

Comment:
Just took another look at your design doc. If I understand it correctly, customers will have to set the credentials themselves? Can we not do that for them? (e.g. construct the authClient for them)

Reply:
They do not have to set the credentials themselves. With the python implementation, but using the signer in the Python SDK, the credentials are automatically read from the environment so that the client doesn't have to provide any parameters to make calls to their API. If they chose to provide credentials to override the signing process, an option will be provided to allow for this.